docs/nidigital/class.rst

```diff
@@ -428,21 +428,23 @@ configure_time_set_compare_edges_strobe
 
     .. py:currentmodule:: nidigital.Session
 
-    .. py:method:: configure_time_set_compare_edges_strobe(pin_list, time_set, strobe_edge)
+    .. py:method:: configure_time_set_compare_edges_strobe(time_set, strobe_edge)
 
             TBD
 
             
 
 
+            .. tip:: This method requires repeated capabilities (pins). If called directly on the
+                nidigital.Session object, then the method will use all repeated capabilities in the session.
+                You can specify a subset of repeated capabilities using the Python index notation on an
+                nidigital.Session repeated capabilities container, and calling this method on the result.:
 
-            :param pin_list:
-
+                .. code:: python
 
-                
+                    session.pins[0,1].configure_time_set_compare_edges_strobe(time_set, strobe_edge)
```

Comment:
Shreyas, what do you think of using something better for this example, like "VCC"?
Should be a separate PR/issue though, and it's low priority.

Reply:
Issue already present - #1125